graphene_sqlalchemy/tests/test_converter.py

```diff
@@ -381,12 +383,28 @@ def test_should_integer_convert_int():
     assert get_field(types.Integer()).type == graphene.Int
 
 
-def test_should_primary_integer_convert_id():
+def test_should_key_integer_convert_id():
```

Comment:
might be worth testing non-default behavior of setting `use_id_type_for_keys` to false too